projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -109,7 +117,16 @@ def default_serialization(o: Any) -> Any:
 
     def _start_job(self, node):
         try:
-            self._job_executor.start_job(node)
+            if (
+                len(self._job_executor.get_currently_running_jobs())
+                > self.max_concurrent_starting_jobs
+            ):
+                log.info(
+                    "Starting job fail - too many concurrently running jobs. Will be re-tried later"
```

Comment:
It would be helpful to log the number of currently running jobs and the maximum allowed number of concurrent jobs in the log message when   delaying a job. This will help in debugging efforts

Reply:
Done.